leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -49,6 +51,7 @@
     <activity
         android:theme="@style/leak_canary_Theme.Transparent"
         android:name=".internal.RequestStoragePermissionActivity"
+        android:process=":leakcanary"
```

Comment:
Thinking about this again, I'm not sure this is worth the effort. These two activities and services aren't large, but also they aren't going to be created concurrently while the leak analysis is occurring.